### ISSUE TITLE
aJax Change on no error form

### ISSUE THREAD

**Issue by Ne-Ne** [jquery-validation/jquery-validation#880]
I have created this code below:

``` js
$("#contactform").validate({
    invalidHandler: function(event, validator){
        $(".send-mssg").removeClass("btn-green").addClass("btn-red");
    },
    validHandler: function(event, validator){
        $(".send-mssg").removeClass("btn-red").addClass("btn-green");
    }
});
```

However when all the label.error, are removed because of a success entry. I want to change the class back. As you can see "btn.red", needs to change to "btn-green" on success, preferably via ajax.

How can this be done?

Any help would be welcome, I have checked through the docs. However I can only see on submit. I want to achieve this prior submit (possible delay?)

-Neil
